docs/ota-client-guide/modules/ROOT/pages/client-provisioning-methods.adoc

```diff
@@ -14,67 +14,80 @@ endif::[]
 :icons: font
 :toc: macro
 
-Before devices can receive updates, each device needs to have a unique identity and a certificate. The provisioning process ensures that a signed certificate is associated with each device. This process is crucial for securing communication between devices and the OTA Connect server.
-OTA Connect supports two provisioning methods. These methods determine the components that will play the role of the "issuer" and the "verifier" in your infrastructure. The "issuer" is the server that signs and issues your device certificates. The "verifier" is the server that verifies the authenticity of device certificates.
-We refer these two methods as "provisioning with shared credentials" and "provisioning with device credentials".
+== What is device provisioning?
 
-* *Provisioning with shared credentials*
-+
-This type of provisioning is great for testing because the OTA Connect server plays the role of both the "issuer" and "verifier" when creating device certificates. You can try out OTA Connect without involving the rest of your infrastructure. However, it's not secure enough for a production scenario.
-+
-* *Provisioning with device credentials*
-+
-This is the provisioning method that you should use in production. In this scenario, the OTA Connect server doesn't issue any credentials to devices; it simply inspects device certificates that come preinstalled. The OTA Connect server only plays the role of "verifier" and the "issuer" role is handled by your fleet certificate authority.
+OTA Connect uses mutual TLS for transport security and device authentication. This means that:
 
-.How the server roles change depending on which method you choose:
-[caption="Figure 1: "]
-image::prov-diff-infra.png[]
+* every device needs to have its own X.509 certificate,
+* every device needs to have a way to trust the X.509 certificate of the OTA Connect server's device gateway, and
+* the OTA Connect server needs to have a way to decide whether it trusts each vehicle's X.509 certificate.
+
+The vehicle knows it can trust the gateway because each OTA Connect account gets a unique device gateway URL, with its own unique X.509 certificate. We then include that certificate in the `credentials.zip` you download, and it gets baked into the image as a pinned certificate authority (CA) when you bitbake.
+
+The device gateway decides whether to allow a device to connect based on the device's X.509 certificate. Every OTA Connect account has one or more *Fleet Root CAs* which are responsible for signing device certificates. When a device connects, your device gateway checks whether its certificate is signed by a trusted Fleet Root CA, and if not, rejects the connection.
+
+*Device provisioning*, therefore, is simply the process of providing a device with:
+
+* an X.509 certificate
+** that is unique to the vehicle
+*** and is signed by a Fleet Root CA that your OTA Connect account trusts
```

Comment:
This gets rendered kind of weird, at least on github. Just wanted to double-check that this is intended.

Reply:
Yes, it was. You think it's too weird?

Reply:
Nah, it's fine, just wanted to confirm the intention.